### DISCUSSION TITLE
Docker wallabag install noob

### DISCUSSION THREAD

**Discussion by cyberwasp65** [wallabag/wallabag#6945] [General]
Not sure if I'm in the right forum or area.  Am new to wallabag and have been unsuccessful in trying to install it in Docker on my Synology nas.  Before I get into any detail, please let me now if I'm in the right place. TIA

**Comment by j0k3r** [comment]
You are in the right place.

**Comment by cyberwasp65** [comment]
Ok.  Below is a copy of  my compose file. I think the line, - SYMFONY__ENV__DOMAIN_NAME=http://192.168.1.25:8082 is the problem but am not sure.

I admit that I have no clue what I'm doing other than following his tutorial.  
version: '3'
services:
  wallabag:
    image: wallabag/wallabag
    environment:
      - MYSQL_ROOT_PASSWORD=wallaroot
      - SYMFONY__ENV__DATABASE_DRIVER=pdo_mysql
      - SYMFONY__ENV__DATABASE_HOST=db
      - SYMFONY__ENV__DATABASE_PORT=3306
      - SYMFONY__ENV__DATABASE_NAME=wallabag
      - SYMFONY__ENV__DATABASE_USER=wallabag
      - SYMFONY__ENV__DATABASE_PASSWORD=wallapass
      - SYMFONY__ENV__DATABASE_CHARSET=utf8mb4
      - SYMFONY__ENV__DATABASE_TABLE_PREFIX="wallabag_"
      - SYMFONY__ENV__DOMAIN_NAME=http://192.168.1.25:8082
      - SYMFONY__ENV__SERVER_NAME="stevebag"
    ports:
      - 8082:80
    volumes:
      - /volume1/docker/wallabag/images:/var/www/wallabag/web/assets/images
    healthcheck:
      test: ["CMD", "wget" ,"--no-verbose", "--tries=1", "--spider", "http://localhost"]
      interval: 1m
      timeout: 3s
    depends_on:
      - db
      - redis
  db:
    image: mariadb
    environment:
      - MYSQL_ROOT_PASSWORD=wallaroot
    volumes:
      - /volume1/docker/wallabag/data:/var/lib/mysql
    healthcheck:
      test: ["CMD", "mysqladmin" ,"ping", "-h", "localhost"]
      interval: 20s
      timeout: 3s
  redis:
    image: redis:alpine
    healthcheck:
      test: ["CMD", "redis-cli", "ping"]
      interval: 20s
      timeout: 3s

**Comment by Kdecherf** [comment]
@cyberwasp65 could you describe the error you encounter?

**Comment by cyberwasp65** [comment]
Well I looked in the logs and there is no mention of an error. However, I just found this:

https://stevenm56.synology.me:6006/sharing/IzBcA82cM

Couldn't find how to attach an image so shared it from nas

**Comment by cyberwasp65** [comment]
version: '3'
services:
  wallabag:
    image: wallabag/wallabag
    environment:
      - MYSQL_ROOT_PASSWORD=wallaroot
      - SYMFONY__ENV__DATABASE_DRIVER=pdo_mysql
      - SYMFONY__ENV__DATABASE_HOST=db
      - SYMFONY__ENV__DATABASE_PORT=3306
      - SYMFONY__ENV__DATABASE_NAME=wallabag
      - SYMFONY__ENV__DATABASE_USER=wallabag
      - SYMFONY__ENV__DATABASE_PASSWORD=wallapass
      - SYMFONY__ENV__DATABASE_CHARSET=utf8mb4
      - SYMFONY__ENV__DATABASE_TABLE_PREFIX="wallabag_"
      - SYMFONY__ENV__DOMAIN_NAME=http://192.168.1.25:8082
      - SYMFONY__ENV__SERVER_NAME="stevebag"
    ports:
      - 8082:80
    volumes:
      - /volume1/docker/wallabag/images:/var/www/wallabag/web/assets/images
    healthcheck:
      test: ["CMD", "wget" ,"--no-verbose", "--tries=1", "--spider", "http://localhost"]
      interval: 1m
      timeout: 3s
    depends_on:
      - db
      - redis
  db:
    image: mariadb
    environment:
      - MYSQL_ROOT_PASSWORD=wallaroot
    volumes:
      - /volume1/docker/wallabag/data:/var/lib/mysql
    healthcheck:
      test: ["CMD", "mysqladmin" ,"ping", "-h", "localhost"]
      interval: 20s
      timeout: 3s
  redis:
    image: redis:alpine
    healthcheck:
      test: ["CMD", "redis-cli", "ping"]
      interval: 20s
      timeout: 3s
